INSTALL.md

```diff
@@ -3,9 +3,10 @@ CVC4 prerelease version 1.9
 
 ## Building CVC4
 
-    ./contrib/get-antlr-3.4  # download and build ANTLR
-    ./configure.sh   # use --prefix to specify a prefix (default: /usr/local)
-                     # use --name=<PATH> for custom build directory
+    ./configure.sh
+        # use --prefix to specify an install prefix (default: /usr/local)
+        # use --name=<PATH> for custom build directory
+        # use --auto-download to download and build missing dependencies
```

Comment:
```suggestion
        # use --auto-download to download and build missing required dependencies
```

Reply:
We also build missing dependencies that are optional, but enabled anyways.

Reply:
I see. Maybe "missing, required or enabled, dependencies"? I think it's somewhat redundant, but useful nonetheless.